getting-started/installation-guide.md

```diff
@@ -181,6 +181,37 @@ Navigate to the UI hostname you configured to login to SystemLink Enterprise as
 
 ## 6. Updating the Application
 
+Updates are done in two steps. The admin release is updated, followed by the application. Upating the admin release is recommended to ensure maximum compatibility with the systemlink application.
+
+### 6.1 Updating Cluster Prerequisites
+
+One of the following types of users must use the systemlinkadmin Helm chart to update prerequisite resources globally on the cluster.
```

Comment:
Do we expect the user to update perquisites every time they update their application? If not, should there be some text about that here?

Reply:
They should only update the prerequisites whenever we release a new version of the chart. I added a comment in the line above.